src/ucs/memory/rcache.inl

```diff
@@ -74,8 +74,7 @@ ucs_rcache_get_unsafe(ucs_rcache_t *rcache, void *address, size_t length,
         }
     }
 
-    return UCS_PROFILE_CALL(ucs_rcache_create_region, rcache, address, length,
-                            prot, arg, region_p);
+    return UCS_ERR_NO_ELEM;
```

Comment:
1. maybe rename this function to `ucs_rcache_`<b>`lookup`</b>`_unsafe`
since `ucs_rcache_`<b>`get`</b> still creates new region

2. return region ptr (which could be NULL if not found)

3. because of this change, can use "if .. return" pattern instead of nested if